pubsub/api/Pubsub.Samples/PullMessagesAsyncWithErrorListen.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START pubsub_subscriber_error_listener]
+
+using Google.Cloud.PubSub.V1;
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+public class PullMessagesAsyncWithErrorListen

```

Comment:
Based on existing sample pattern, I think the class should be named `PullMessagesWithErrorListenerAsyncSample` and file should be named `PullMessagesWithErrorListenerAsync.cs`